### ISSUE TITLE
Inconsistent indentation not triggered

### ISSUE THREAD

**Issue by royaldark** [clutchski/coffeelint#485]
Why does this trigger the inconsistent indentation error `(expected 2, got 1)`:
```coffeescript
->
 a
 b
```
...but this does not?
```coffeescript
->
  a
 b
```

The second is actually far worse, since it compiles to valid but unexpected JS:
```javascript
(function() {
  return a;
});

b;
```

I'm running the newest version, 1.10.1.

**Comment by swang** [comment]
Unfortunately this is due to CoffeeScript's parser. It does not return an INDENT token for `b` statement, rather it gives an OUTDENT 1 statement.

Since spacing of 1 space is allowed in CoffeeScript, it assumes you are indenting by one space, and that the `b` statement is just a outdent of 1, rather than an indentation of 1 (which would trigger inconsistent indentation rule)

I will try to take a look at this bug, as it may be easier to catch than the other INDENT issues we've had. Thanks for the report.

**Comment by swang** [comment]
Hello. Basically what is happening can probably be explained in this issue jashkenas/coffeescript#3998. I don't know if this is really fixable without getting some help from the CoffeeScript developers.
